servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcRouter.java

```diff
@@ -271,13 +269,10 @@ private static void mergeRoutes(final Map<String, RouteProvider> first,
                                                         serializationProvider.serializerFor(responseEncoding,
                                                                 responseClass)))
                                         .recoverWith(cause -> {
-                                            LOGGER.error("Unexpected exception from route: {}, path: {}.", route, path,
```

Comment:
I liked the logging inside the `recoverWith` operator. it gives path/route info and it can be muted if it causes noise.

Reply:
Or we can log it only for unexpected exceptions, but ignore for `GrpcStatusException` because only user can generate `GrpcStatusException` before control flow reaches this place.

Reply:
I would prefer to avoid putting debug logging in the core that can be added at higher layers. if more context is required users can always apply their own error recovery to log, record metrics, etc.